extensions/wikia/AchievementsII/AchConfig.class.php

```diff
@@ -531,51 +524,50 @@ public function getTrackNameKey($badgeTypeId) {
 		}
 	}
 
-	public function getBadgeTrackCategory($badgeTypeId) {
-		if($badgeTypeId < 0)
+	public function getBadgeTrackCategory( $badgeTypeId ) {
+		if( $badgeTypeId < 0 )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add curly braces around the nested statement(s). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS121)